### PR TITLE
Adjust breadcrumbs for organizations

### PR DESCRIPTION
**Pull request by FrankeLukas**
### Checklist
- [x] I tested *all* changes and *all* related features with different users (student, tutor, editor, instructor, admin) on the test server https://artemistest.ase.in.tum.de.
- [x] Server: I followed the [coding and design guidelines](https://docs.artemis.ase.in.tum.de/dev/guidelines/server/).
- [x] Server: I added multiple integration tests (Spring) related to the features (with a high test coverage)
- [x] Server: I added `@PreAuthorize` and check the course groups for all new REST Calls (security)
- [x] Server: I implemented the changes with a good performance and prevented too many database calls
- [x] Server: I documented the Java code using JavaDoc style.
- [x] Client: I followed the [coding and design guidelines](https://docs.artemis.ase.in.tum.de/dev/guidelines/client/).
- [x] Client: I added multiple integration tests (Jest) related to the features (with a high test coverage)
- [x] Client: I added `authorities` to all new routes and check the course groups for displaying navigation elements (links, buttons)
- [x] Client: I documented the TypeScript code using JSDoc style.
- [x] Client: I added multiple screenshots/screencasts of my UI changes

### Motivation and Context

There were no breadcrumbs in the organization management other than for the overview.

### Description

Now each organization will display the title as breadcrumb (and the edit page an edit crumb).

### Steps for Testing

1. Navigate to Organization Management (as admin)
2. Open an organization and check that the title is displayed
3. Edit an organization and check that a "Edit" crumb is displayed
4. Open the creation page and check that a "Create" crumb is displayed.

### Review Progress

- Code Review
  - [x] Review 1
  - [x] Review 2
- Manual Tests
  - [x] Test 1
  - [x] Test 2

### Screenshots

grafik

grafik

### CODE REVIEW

File: src/main/webapp/app/shared/layouts/navbar/navbar.component.ts [hunk 1]
```diff
@@ -282,6 +284,9 @@ export class NavbarComponent implements OnInit, OnDestroy {
                 this.routeExamId = Number(segment);
                 this.addResolvedTitleAsCrumb(this.examService.getTitle(this.routeExamId), currentPath, segment);
                 break;
+            case 'organisation-management':
+                this.addResolvedTitleAsCrumb(this.organisationService.getTitle(Number(segment)), currentPath, segment);
+                break;
```
[the remark this answers is not in the document]
Comment (FrankeLukas) [reply src/main/webapp/app/shared/layouts/navbar/navbar.component.ts:289]: I'll likely add a test, that never hurts.

**Review by FuchsDominik — APPROVED** [review]
Code looks fine 👍

**Review by jhainzlschmid — APPROVED** [review]
Tested on ts3. Breadcrumbs are added when viewing, editing and creating organizations.
Code looks good as well 👍

**Review by coolchock — APPROVED** [review]
code looks good 👍

**Review by Olimbia23 — APPROVED** [review]
Thank you for adding the test @FrankeLukas.
Approve 👍

**Pull request merged**